lightpath/ui/gui.py

```diff
@@ -44,10 +48,14 @@ class LightApp(Display):
 
     parent : optional
     """
-    shown_types = [dtypes.Attenuator, dtypes.GateValve, dtypes.IPM,
-                   dtypes.LODCM, dtypes.OffsetMirror, dtypes.PIM, PPSStopper,
-                   dtypes.PulsePicker, dtypes.Slits, dtypes.Stopper,
-                   dtypes.XFLS]
+    shown_types = {'Attenuator': AttBase, 'Gate Valve': dtypes.GateValve,
```

Comment:
`isinstance` can check against tuples of types, so we could include tuples of types in the dict here to catch cases where multiple unrelated type trees represent the same "kind" of object.

Alternatively, maybe it's time to rework pcdsdevices to include bare `AttenuatorType` base classes with no code in them.

Reply:
I'd be OK with that or just type-hint based ones: `AttenuatorType = Union[AttBase, ...]`

Reply:
This could be dynamically generated based on the device instances that are currently shown, though it wouldn't be as nice as a manually curated list.

Reply:
Zach and I settled on taking the module name as the "device type", which simplifies the work here and helps incorporate future changes more easily.  

Under this scheme, we'd group all devices from `pcdsdevices.attenuators` under the same checkbox, everything under `pcdsdevices.pim` together, etc

(I'll commit when I'm able, but do you have any objections to this approach?)

Reply:
Sounds good to me 👍